docs/reference/semaphore-changelog.md

```diff
@@ -8,6 +8,33 @@ Thank you for using Semaphore!
 We continuously deploy changes that improve the product for you.
 This page is updated on a weekly basis.
 
+### Week of January 4, 2021
+**(Improved) Ubuntu 18.04 image update**  
+**(New)** Language versions:  
+
+- Ruby 3.0.0
+- Php 8.0.0
+
+**(New)** Package:
+
+- [Shellcheck](https://packages.ubuntu.com/bionic/shellcheck)
+
+**(Updated)** Updated packages:
+
+- Azure-cli 2.15.1 -> 2.17.1
+- Aws-cli 1.18.183 -> 1.18.207
+- Docker 19.0.3 -> 20.10.1
+- Erlang 23.1 -> 23.2.1
+- Git 2.29 -> 2.30
+- Git-Lfs 1.12.1 -> 1.13.1
+- Heroku 7.47.3 -> 7.47.6
+- Pypy 7.3.2 -> 7.3.3
+- Pip 20.2 -> 20.3.3
```

Comment:
Pip version is fixed to 20.2.4 now.

Reply:
The file still shows the old version but I will update it along with the ubuntu image page. 
@bogyo210 anything that needs to be added for erlan?

Reply:
Nope. 23.2 is the latest one, 23.1 will be removed from the image, still present because of nedap needs it. The changelog should state the update and I'll leave 23.1 until February so the update will be public after you merge, in February no one should complain.

Reply:
@bogyo210 I have changed pypy and pip version in ubuntu 18.04 image page to reflect the changes described in changelog.